frontend/src/components/navbar/Navbar.tsx

```diff
@@ -1,31 +1,67 @@
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import { Link } from "react-router-dom";
 import { LoginContext } from "../login/LoginManager";
 import "./Navbar.scss";
+import Hamburger from "hamburger-react";
 
 export const Navbar: React.FunctionComponent = () => {
   const loginContext = useContext(LoginContext);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  function toggleMenu() {
+    if (isExpanded) {
+      setIsExpanded(false);
+    }
+  }
 
   return (
     <nav className="navbar" role="navigation" aria-label="main navigation">
+      <div className="mobile-menu">
+        <Hamburger toggled={isExpanded} toggle={setIsExpanded} />
+      </div>
       <Link to="/">
         <img
           className="navbar__logo"
           src="/logo.png"
           alt="Whale Spotting logo"
         />
       </Link>
-      <div>
-        {!loginContext.isLoggedIn ? (
-          <div></div>
-        ) : (
-          <Link to="/">
-            <a className="button is-primary" onClick={loginContext.logOut}>
-              <strong>Logout</strong>
-            </a>
-          </Link>
-        )}
-      </div>
+      <ul className={`menu-items ${isExpanded ? "expanded" : ""}`}>
+        <Link to="/" onClick={() => toggleMenu()}>
+          <h1>WHALESPOTTING</h1>
+        </Link>
+        <Link to="/browse-sightings" onClick={() => toggleMenu()}>
+          <h1>Sightings</h1>
+        </Link>
+        <Link to="/whaleopedia" onClick={() => toggleMenu()}>
+          <h1>Whaleopedia</h1>
+        </Link>
+        <Link to="/videos" onClick={() => toggleMenu()}>
+          <h1>Videos</h1>
+        </Link>
+
+        <div className="admin-links">
+          {!loginContext.isLoggedIn ? (
+            <Link to="/login" onClick={() => toggleMenu()}>
+              <h1>Log On</h1>
+            </Link>
+          ) : (
+            <div className="admin-only-links">
+              <Link to="/sightings/pending" onClick={() => toggleMenu()}>
+                <h1>Pending</h1>
+              </Link>
+              <Link to="/create-user" onClick={() => toggleMenu()}>
+                <h1>User+</h1>
+              </Link>
+              <Link to="/" onClick={() => toggleMenu()}>
+                <a className="button is-primary" onClick={loginContext.logOut}>
+                  <h1>Log Out</h1>
+                </a>
+              </Link>
```

Comment:
`<Link>` components automatically use an `<a>` tag, so you should never put another `<a>` inside a `<Link>` or you'll end up with two `<a>`s on the page!

But, in this case, the "log out" functionality shouldn't necessarily redirect the user (it obviously will if the user is on an admin-only page, but if the user is on a normal page like the video page they shouldn't be sent away to the homepage when they log out!)

Instead, since this isn't a navigation link but just other functionality, you should make it a `<button>` and use CSS to make it _look_ like the other links. It's a bit weird, but the semantics match up!

```suggestion
              <button onClick={() => {
                toggleMenu();
                loginContext.logOut();
              }>
                Log Out
              </button>
```